api/managementpb/ia/channels.proto

```diff
@@ -24,6 +24,9 @@ message TLSConfig {
   string key_file = 3;
   string server_name = 4;
   bool insecure_skip_verify = 5;
+  string ca_file_content = 6;
+  string cert_file_content = 7;
+  string key_file_content = 8;
```

Comment:
please add comments to these fields and fields above.
What is the difference between ca_file and ca_file_content?

Could you share doc or post how to use `ca_file`, `cert_file` and e.t.c.

Reply:
comments added